readme.md

```diff
@@ -128,6 +128,14 @@ If you want to install a plugin from Kirby's official Plugins organisation (<htt
 kirby plugin:install cachebuster-plugin
 ```
 
+#### Branch Option
+
+You can pass a `--branch=dev`-Option to install the plugin form a specific branch. It defaults to the master branch.
```

Comment:
Small typo: *from* a specific branch. I also think we should make more clear that it works with any branch.